src/lib/terra-components.module.ts

```diff
@@ -55,7 +55,8 @@ import { MatTreeModule } from '@angular/material/tree';
         MatPseudoCheckboxModule,
         MatToolbarModule,
         MatExpansionModule,
-        MatTreeModule
+        MatTreeModule,
+        MatPseudoCheckboxModule
```

Comment:
This is already added. See L55.
Didn't you revert using git?

Reply:
I reverted the command `Merge pull request...` via GHD .

Reply:
Seems like reverting Merges that had conflicts is a mess 🙈

Reply:
but please remove this duplicate import

Reply:
and i think the expansion module import must be removed

Reply:
done

Reply:
Why the MatExpansionModule? That's not case from the image preview and maybe needed? in schematic maybe? 😅

Reply:
it is only used in the image preview